### PR TITLE
Create recommendations from Ning for Beijing

### LINKED ISSUE #13: Create a new file for lengning's hometown (Beijing China)
## Beijing, China
- [x] create a branch
- [x] commit
- [ ] open a pull request
- [ ] review and collaboration
- [ ] seek approval
- [ ] merge

### PR DESCRIPTION
**Pull request by lengning**
closes #13

### CODE REVIEW

File: places/lengning-beijing.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Subway in rush hours
+
+### Places to go
+
+Many
+
+- summer palace
+
+- forbidden city
```
Comment (pourmoht) [inline places/lengning-beijing.md:17]: How can I go if the city is forbidden!? 😕

File: places/lengning-beijing.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## Beijing China
+
+### Places to eat
+
+Gui Street (downtown)
+
+### Places to avoid
+
+Subway in rush hours
```
Comment (pourmoht) [inline places/lengning-beijing.md:9]: Are 🚗 's a popular choice?

Comment (lengning) [reply places/lengning-beijing.md:9]: tons of cars too 😿

File: places/lengning-beijing.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+### Places to go
+
+Many
+
+- summer palace
```
Comment (pourmoht) [inline places/lengning-beijing.md:15]: Is there a winter palace? WINTER IS COMING!!!

**Review by pourmoht — APPROVED** [review]

**Pull request merged**